zkg.meta

```diff
@@ -10,6 +10,6 @@ depends =
   zeek >=4.0.0
   zkg >=2.0
 external_depends =
```

Comment:
I don't understand why it is 1.4.4 other places and 1.4.2 here?

Reply:
It's `~1.4.2` meaning we depend on this version or anything greater in the same minor range (i.e. 1.4.x).  We support as old as 1.4.2, or newer, but our tests run against 1.4.4.  That's why I made sure it was still 1.4.2 in 20381521089051ab7ee4dd0ac2ad887228587b79

Reply:
but we want it less than the latest?

Reply:
@ottobackwards the latest release failed our tests out of the box so I reverted to this patch update for the time being